packages/common/src/types/filter.ts

```diff
@@ -66,12 +69,18 @@ export type DashboardFieldTarget = {
     tableName: string;
 };
 
+export type TileFieldTargetOverride = DashboardFieldTarget & {
+    tileUuid: string;
+};
+
 export type DashboardFilterRule<
```

Comment:
@owlas it's still relevant, because not all tiles override the default target. we can remove the `target` and `tileTargetOverride` if we replace it with `targets: Record<string, DashboardFieldTarget>` but then we need to have a migration for backward compatibility or do the ad-hoc magic in the frontend. thoughts?